TripPlaner/src/main/resources/templates/cityAddForm.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet">
+  <link href="https://getbootstrap.com/docs/5.3/assets/css/docs.css" rel="stylesheet">
+  <title>Singapore - create City</title>
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js"></script>
+</head>
+<body class="p-3 m-0 border-0 bd-example">
+<div action="#" th:action="@{/city}" th:object="${city}" method="post">
+<form class="row g-3" >
+  <div class="col-md-6">
+    <label for="inputCityName" class="form-label">Nazwa miasta:</label>
+    <input type="text" class="form-control" id="inputCityName" required th:field="*{name}">
+  </div>
+
+  <div class="col-12">
+    <label for="Textarea" class="form-label">Opis miasta:</label>
+      <input type="text" class="form-control" id="Textarea" required th:field="*{description}">
+  </div>
+
+  <div class="col-md-4">
+    <label for="inputCountry" class="form-label">Kraj</label>
+    <input type="text" class="form-control" id="inputCountry">
+  </div>
+  <div class="col-md-4">
+    <label for="inputContinent" class="form-label">Kontynent</label>
+    <select id="inputContinent" class="form-select">
+      <option selected="">Wybierz...</option>
+      <option>Europa</option>
+      <option>Azja</option>
+      <option>Afryka</option>
+      <option>Ameryka Południowa</option>
+      <option>Ameryka Północna</option>
+      <option>Australia i Oceania</option>
+      <option>Antarktyda</option>
+    </select>
```

Comment:
Powinno się znaleźć w modelu obiektu i być generyczne jeśli chcecie tego używać.

Reply:
Usunę to i w miejsce tego proponuję wrzucić mapę, która będzie zwracać współrzędne geograficzne po kliknięciu.